blog/2023-07-27-introducing-substratus.md

```diff
@@ -0,0 +1,137 @@
+   each resource share considerable commonality so there's little to keep in
+   your head.
+2. **Limited integration points with our supported cloud providers**: We are as
+   incentivized as you to avoid unnecessary cloud spend. To this end, we use
+   strictly commodity services with open-source on-prem equivalents or analogs
+   (e.g., S3, GCS, ABS, ceph, minio, etc.).
+3. **Infrastructure-primitives serve the ends of first-class devX**: Rock-solid
+   infrastructure is our bread and butter and isn't negotiable for a project
+   like Substratus but infra is also rarely sufficient to delight users. The
+   ability to chain Substratus primitives through high-level clients is where we
+   believe the s8s starts to feel like magic.
+
+## Our Product Roadmap
+
+There's no shortage of features we plan to build and we have no intention of
+keeping our cards close. We hope to make the days of ML engineers, cluster
+operators, and security teams just a little better at every step in our journey.
+Here's what we have on-deck:
+
+1. **Supporting each of the big-3 cloud platforms**: As of now, we support
+  Google Cloud Platform, with AWS and Azure support coming soon.
+2. **VSCode Integration**: We're designing and developing a VSCode extension to
+  bring a first-class notebook experience into your favorite editor. The Jupyter
+   Extension for VSCode is a great foundation for that experience today, but we
+   see room for another level of excellence by knitting infrastructure
+   operations into a higher-level, developer experience and workflow.
+3. **Large-scale distributed training**: We aim to support the largest of
+  training workloads. Currently we're able to utilize 8 GPUs for a single model
+  but we've heard this isn't enough for every use-case. We intend to make wider
+  horizontal GPU distribution a paved path on Substratus.
+4. **Partner Integrations**: We're teaming up with our friends at LangChain and
+   Weaviate to provide integrations and demos showcasing how and where
+   Substratus fits in the larger ML ecosystem.
+
+The roadmap above is all high-priority for our open-source project but we think
+there's great value in getting started at low or no-cost without, and without
+standing up any infrastructure of your own. These upcoming features form the
+foundation of our fledgling SaaS product:
+
+1. **Substratus Cloud**: Soon, you'll be able get an API key and unlock the same
+   Substratus open-source API through a managed platform! Forget the quotas of
+   your personal cloud account, our SaaS is the perfect way to get started and
+   share your work without deploying infrastructure.
+2. **RBAC for Teams**: Role-based access control for teams, ensuring security
+   and privacy for your projects.
+3. **Multi-cloud, multi-cluster operations**: Manage your hybrid-SaaS Substratus
+  clusters easily without needing to switch kubecontexts. Our enterprise CLI is
```

Comment:
Is this how y'all envision the split in functionality being exposed? Maybe it's not important to get right here but I don't think we'll ship a separate enterprise cli

Reply:
I would probably remove this part about enterprise CLI